modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/DataRegionMetricsImpl.java

```diff
@@ -726,6 +726,11 @@ public void clear() {
             metrics.reset();
     }
 
+    /** Removes all metric for data region. */
+    public void remove() {
+        kernalCtx.metric().remove(metricRegistry().name(), false);
```

Comment:
Can we create dedicated variable for the registry name and calculate it once?

Reply:
I think code should not be refactored in this issue.